### ISSUE TITLE
[BUG] Accessibility: Disable Input labels and hint text fail WCAG 1.4.3 (Rest of components)

### ISSUE THREAD

**Issue by lstevens98** [UKHO/admiralty-design-system#291]
Additional task to fix the rest of the components with disabled input labels and hint text that are failing WCAG 1.4.3.

See original ticket #160 (https://github.com/orgs/UKHO/projects/40/views/1?pane=issue&itemId=52553988&issue=UKHO%7Cadmiralty-design-system%7C160) for details.

**Comment by developernm** [comment]
Need to confirm whether the checkbox needs updating as when its disabled, it looks active. 
@KatiePUX what are your thoughts on this?

![Image](https://github.com/user-attachments/assets/20c21b68-50fb-4348-a266-01e72f281e1b)

**Comment by developernm** [comment]
Fixed disabled checkbox.

![Image](https://github.com/user-attachments/assets/b7f7aa31-b25d-47b0-b06b-2e5046d4a2d9)

**Comment by developernm** [comment]
Closing as its done
